docs/content/docs/KnownIssues.md

```diff
@@ -11,3 +11,14 @@ type: docs
 Disable **Settings > Experimental > Only sync the active variant**
 
 https://stackoverflow.com/questions/54201216/android-studio-3-3-new-gradle-sync-is-not-supported-due-to-containing-kotlin-mo/
+
+## Tests run from Android Studio are failed in runtime with `java.lang.NoClassDefFoundError`
+
+The problem occurs only when running a test from Android Studio because of wrongly generated Run Configuration.
```

Comment:
"wrongly generated"

It's better to add an explanation of the exact problem: configuration has an inappropriate type.
For me, this information is the key. When I understand what is correct and what is not, I can easily check it.

Reply:
I've added it. Thanks.